### PR TITLE
Use pip's single-source version scheme

### PR DESCRIPTION
**Pull request by jfolz**
Avoid additional files and magic functions to extract the package version during setup.

### CODE REVIEW

File: pyre/__init__.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 __all__ = ['pyre', 'zbeacon', 'zhelper']
-
-from pkg_resources import resource_string
-__version__ = resource_string('pyre', 'VERSION')
+__version__ = '0.3.2'
+__version_info__ = tuple(int(v) for v in __version__.split('.'))
```
Comment (jfolz) [inline pyre/__init__.py]: Not sure if parts should be int or str.

**Comment by jfolz** [comment]
No idea why that travis issue in PY32 popped up just now.

**Comment by jfolz** [comment]
There is also this warning:
`/home/travis/virtualenv/python3.2.6/lib/python3.2/site-packages/pkg_resources/__init__.py:82: UserWarning: Support for Python 3.0-3.2 has been dropped. Future versions will fail here.`

File: pyre/__init__.py (hunk 1 above)
Comment (keent) [inline pyre/__init__.py]: When does it matter?

**Comment by keent** [comment]
Thanks

File: pyre/__init__.py (hunk 1 above)
Comment (jfolz) [inline pyre/__init__.py]: Depends on what we expect the version method to return.
Also can't have versions like `0.3.2-dev` with ints.

Comment (keent) [inline pyre/__init__.py]: This is more than enough for now. Thanks

**Comment by papr** [comment]
Thanks @jfolz I like this implementation much better than mine, good work!

**Pull request merged**